src/components/RecommendationCount/RecommendationCount.tsx

```diff
@@ -15,6 +15,7 @@ type Props = {
     alreadyRecommended: boolean;
     isSignedIn: boolean;
     userMadeComment: boolean;
+    onRecommend: (commentId: number) => Promise<Boolean>;
```

Comment:
Would this have to be an optional prop in order for dotcom to ignore it and have a fallback to the default?

```suggestion
    onRecommend?: (commentId: number) => Promise<Boolean>;
```